src/no_std.rs

```diff
@@ -576,7 +576,7 @@ impl<T> ListenerSlab<T> {
 
             // Notify the next entry.
             match self.start {
-                None => break,
+                None => return original_count - n - 1,
```

Comment:
The "Fix off-by-one error in implementation" commit seems to be a fixup commit for the first one. If that's the case, please squash it into that and force push. I am a champion of atomic commits but that's more about each commit keeping only one logical commit rather than immutability of the commits.